### PR TITLE
Fixes meteors bumping in nullspace causing runtimes.

### PR DESCRIPTION
**Pull request by gbasood**
I literally have no idea on another solution for this
It's not ideal but it works

### CODE REVIEW

File: code/game/gamemodes/meteor/meteors.dm [hunk 1]
```diff
@@ -170,7 +171,8 @@
 	icon_state = "medium_radioactive"
 
 /obj/effect/meteor/radioactive/Bump(atom/a)
-
+	if(loc == null)
```
Comment (ghost) [inline code/game/gamemodes/meteor/meteors.dm:174]: Couldn't you just have each child of meteor inherit the bump from the parent object, save copypasting the check, or does that not work with effects?

Comment (Intigracy) [inline code/game/gamemodes/meteor/meteors.dm:174]: it already inherits, but it inherits at the end of the proc.

Like this one applies radiation to people in range before calling meteor/bump.

Comment (gbasood) [inline code/game/gamemodes/meteor/meteors.dm:174]: Exactly

**Comment by clusterfack** [comment]
Did you see what happens if you call Move() in nullspace, whether it bumps or not?

**Comment by clusterfack** [comment]
Also try putting a message into bump specifically and see if its being called twice, if it is something could be very wrong with our move code.

**Comment by gbasood** [comment]
@clusterfack I did the second. It Is being called twice.
I had a to_chat at the top of the flash meteor's Bump() that only fired if loc was null, and the meteor exploded before that to_chat was reached, but it absolutely was.

I didn't try the first.

**Pull request merged**